docs/send-data/opentelemetry-collector/remote-management/source-templates/st-with-secrets.md

```diff
@@ -0,0 +1,97 @@
+---
+id: st-with-secrets
+title: Setting Environment Variables with Secret Values for Source Templates
+sidebar_label: Setting Environment Variables
+description: Steps for setting environment variable with secret value which can be used by source template at runtime in a remotely managed OpenTelemetry collector.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
```

Comment:
Since this doc is present in the sidebars, I'd remove this code snippet, which is only intended for beta and/or hidden docs. It blocks it from Algolia and Google search.

Reply:
@sumoanema